ui/src/router/index.js

```diff
@@ -40,7 +40,17 @@ router.beforeEach((to, from, next) => {
     } else {
       next();
     }
-  } else {
+  }
+  else if (to.path=="/login")
```

Comment:
The double equal works here, but it is best practice to use `===` to check for strict equality. It should also be between spaces.